terraform/hydrocron-iam.tf

```diff
@@ -135,6 +135,41 @@ data "aws_iam_policy_document" "lambda_log_to_cloudwatch" {
   }
 }
 
+data "aws_iam_policy_document" "sqs-resource-policy" {
```

Comment:
Is the SQS permissions still needed if an SQS queue is no longer used? Just want to make sure I am not missing something in the updates to the database code!

Reply:
Thanks NIkki, it's not needed, looks like I just failed to completely remove it. Will do so now